spring-boot-dependencies/pom.xml

```diff
@@ -158,7 +158,7 @@
 		<spring-data-releasetrain.version>Kay-M3</spring-data-releasetrain.version>
 		<spring-hateoas.version>0.23.0.RELEASE</spring-hateoas.version>
 		<spring-integration.version>5.0.0.BUILD-SNAPSHOT</spring-integration.version>
-		<spring-kafka.version>2.0.0.M1</spring-kafka.version>
+		<spring-kafka.version>2.0.0.BUILD-SNAPSHOT</spring-kafka.version>
```

Comment:
Can't merge this until we get M2 I suppose? We are releasing a milestone next week.